app/webpack/javascripts/components/tooltip.js

```diff
@@ -0,0 +1,6 @@
+require('../../stylesheets/application.scss');
```

Comment:
Why exactly is the application.scss required in the tooltip, which is plain bootstrap and JS only?